config/initializers/omniauth.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
   provider :facebook, ENV['FACEBOOK_KEY'], ENV['FACEBOOK_SECRET'],
-            scope: 'public_profile', 
-            info_fields: 'id,name,link'
+            scope: 'public_profile,email', 
+            info_fields: 'id,name,link,email,location,significant_other,hometown'
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.